integrationtest/performance/orm/src/test/java/org/hibernate/search/test/performance/TestRunnerArquillian.java

```diff
@@ -58,7 +57,6 @@
 				.addPackages( true, TestRunnerArquillian.class.getPackage() )
 				.addClass( TestConstants.class )
 				.addAsResource( createPersistenceXml(), "META-INF/persistence.xml" )
-				.addAsLibraries( PackagerHelper.hibernateSearchTestingLibraries() )
```

Comment:
Not to be adressed in this ticket, but we also seem to be using two Hibernate Search JARs in other tests:

* `MemberRegistrationEarArchiveIT`: we add one as library, and also define `wildfly.jpa.hibernate.search.module`
* `CDIInjectionIT`: we using the `hibernateSearchTestingLibraries`, which depend on Hibernate Search. It's convenient, but we could probably avoid any JAR conflict by adding the `hibernate-search-engine` and `hibernate-search-orm` JARs (at least) to the transitive dependencies exclusions for the testing libraries.

Should I create a ticket for that?

Reply:
Nice catch, I'm having a look.

Reply:
Addressed by PR #1550